map_gui/src/tools/minimap.rs

```diff
@@ -101,38 +101,35 @@ impl<A: AppLike + 'static, T: MinimapControls<A>> Minimap<A, T> {
         let zoom_col = {
             let mut col = vec![ctx
                 .style()
-                .btn_plain_icon("system/assets/speed/speed_up.svg")
+                .btn_plain_icon("system/assets/speed/plus.svg")
                 .build_widget(ctx, "zoom in")
                 .centered_horiz()
-                .margin_below(20)];
+                .margin_below(10)];
+
+            let level_btn = ctx
+                .style()
+                .btn_plain_icon("system/assets/speed/zoom_level_rect.svg")
+                .padding_top(0.0)
+                .padding_bottom(0.0);
+
             for i in (0..=3).rev() {
-                let color = if self.zoom_lvl < i {
-                    app.cs().minimap_unselected_zoom
+                let level_btn = if self.zoom_lvl < i {
+                    level_btn
+                        .clone()
+                        .image_color(ctx.style().btn_outline.fg_disabled, ControlState::Default)
                 } else {
-                    app.cs().minimap_selected_zoom
+                    level_btn.clone()
                 };
-                let rect = Polygon::rectangle(20.0, 8.0);
-
-                let default_batch = GeomBatch::from(vec![(color, rect.clone())]);
```

Comment:
This was a cute trick, but ultimately I thought it looked better to style the "zoom increments" like a normal button, and the easiest way to do that was to make a simple svg rectangle.

Reply:
The less custom buttons, the better!